dumpMetadata: disable signature validation

Makes it possible to work on RPM repositories that contain signed packages
without the need of importing the public part of the signing key into the RPM
database.

Upstream-Status: Pending

Signed-off-by: Markus Lehtonen <markus.lehtonen@linux.intel.com>
---
 dumpMetadata.py | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/dumpMetadata.py b/dumpMetadata.py
index e40e8ac..ca6a82d 100644
--- a/dumpMetadata.py
+++ b/dumpMetadata.py
@@ -92,7 +92,9 @@ def returnHdr(ts, package):
             fdno = package # let's assume this is an fdno and go with it :)
     except OSError:
         raise MDError, "Error opening file"
-    ts.setVSFlags((rpm.RPMVSF_NOMD5|rpm.RPMVSF_NEEDPAYLOAD))
+    ts.setVSFlags((rpm.RPMVSF_NOMD5|rpm.RPMVSF_NEEDPAYLOAD|
+                   rpm.RPMVSF_NODSA|rpm.RPMVSF_NORSA|
+                   rpm.RPMVSF_NODSAHEADER|rpm.RPMVSF_NORSAHEADER))
     try:
         hdr = ts.hdrFromFdno(fdno)
     except rpm.error:
-- 
2.1.4

